### DISCUSSION TITLE
can I get interval time when I config the refetchInterval?

### DISCUSSION THREAD

**Discussion by lili21** [TanStack/query#8290] [General] [answered]
I want to display a count down timer to indicate user when the data will refetch. It would be nice if I can get the value from `useQuery`.

**Comment by TkDodo** [comment, marked as answer]
we don’t expose that because we only schedule a `setTimeout` to trigger the next fetch in N seconds. We also can’t even determine at which granularity that should be exposed. Update once every second? Once every couple of seconds? Ten times a second?

If you want that, you have to do it yourself in user-land.
